modules/display.c

```diff
@@ -6935,25 +6998,16 @@ static gboolean mdy_dbus_send_blanking_inhibit_status(DBusMessage *const method_
      */
     if( display_state == MCE_DISPLAY_ON ||
         display_state == MCE_DISPLAY_DIM ) {
-            curr = !(mdy_blanking_off_cb_id || mdy_blanking_dim_cb_id);
-
+        if( !mdy_blanking_off_cb_id && !mdy_blanking_dim_cb_id )
+            curr = true;
```

Comment:
This is something I struggled with when writing the code. Now the odd indent brought it to my eyes and I decided that there is nothing to like about the "not (a or b)" construct + the fact that it overwrites the value instead of switching to true when applicable. And yes, it should have been a separate commit.